Kanban/Controllers/KBNAddTaskViewController.m

```diff
@@ -12,10 +12,11 @@
 #import "KBNTaskService.h"
 #import "UITextView+CustomTextView.h"
 
-@interface KBNAddTaskViewController ()
+@interface KBNAddTaskViewController () <MBProgressHUDDelegate>
 
```

Comment:
Same as before, declare that your class adopts the protocol in the class definition (.h)